lib/go-atscfg/strategiesdotconfig.go

```diff
@@ -182,6 +191,9 @@ func getStrategyStrategiesSection(pa *ParentAbstraction) string {
 			txt += "\n" + `    hash_key: path`
 		}
 		txt += "\n" + `    go_direct: ` + strconv.FormatBool(svc.GoDirect)
+		if getStrategySecondaryMode(svc.SecondaryMode) == "peering_ring" {
```

Comment:
Nitpick: `"peering_ring"` should probably be put in a constant somewhere. Probably `lib/go-atscfg` so other things can use it